manual/manual.tex

```diff
@@ -307,6 +314,25 @@ \section{Runtime Configuration\label{runtime-config}}
 yield point, instead of sampling the time. This has a higher runtime
 overhead than \texttt{velocity} and we do not recommend it.
 
+\subsection{Stack behavior \texttt{(.stackSize and .restoreFrames)} }
+Stopify supports heap bounded stacks for programming languages that want to
+support deeply recursive code and not inherit the stack size limit imposed
+by JavaScript VMs. Stopify accomplishes by restoring only
```

Comment:
I find the prefix/suffix terminology a little weird coming from the perspective of someone who doesn't know how Stopify represents things internally. I would maybe rewrite this part of the docs to refer to Stopify as reifying stacks on the heap and restoring stack frames lazilly and avoid talking about "once a program finishes running, Stopify checks...".